cmd/juju/commands/run.go

```diff
@@ -102,9 +105,13 @@ func (c *runCommand) SetFlags(f *gnuflag.FlagSet) {
 	})
 	f.BoolVar(&c.all, "all", false, "Run the commands on all the machines")
 	f.DurationVar(&c.timeout, "timeout", 5*time.Minute, "How long to wait before the remote command is considered to have failed")
+	f.DurationVar(&c.timeout, "t", 5*time.Minute, "How long to wait before the remote command is considered to have failed")
```

Comment:
The code would be better like this:

for _, flag := range []string{"timeout", "t"} {
    f.DurationVar(&c.timeout, flag, 5*time.Minute, "How long to wait before the remote command is considered to have failed")
}

etc (similar for unit and application)